pdm/init/action.yml

```diff
@@ -47,10 +47,10 @@ runs:
         pdm config install.cache true
       shell: bash
 
-    - name: Install dependencies
+    - name: Install dependencies (main, all extras and specified dev group)
       run: |
         echo "${{ github.workspace }}/.venv/bin/python" > .pdm-python
-        pdm sync --dev --group ${{ inputs.group }}
+        pdm sync --group :all --dev --group ${{ inputs.group }}
```

Comment:
`--group :all` installs all deps right?
Is it useful to have again the `--group ${{ inputs.group }}` ?